integration-test/lolex-integration-test.js

```diff
@@ -57,3 +57,83 @@ describe("withGlobal", function () {
         clock.uninstall();
     });
 });
+
+describe("globally configured browser objects", function () {
+    var withGlobal, originalDescriptors;
+
+    // Used to support Node 6 which doesn't support Object.getOwnPropertyDescriptors
```

Comment:
I'll try to fix the issues in #87 so this won't be needed.

But thank you for good comments!